### ISSUE TITLE
[BUG] Can't mark reading activity as completed

### ISSUE THREAD

**Issue by BTWS2** [thepieterdc/dodona-plugin-vscode#569]
# Description
1. Mark reading activity as completed in sidebar (see screenshot at the end)
2. Confusing message:
![image](https://user-images.githubusercontent.com/56451049/217266873-cfd35010-1be0-4b34-8e0f-8eece994d1d2.png)
3. Go to reading activity in Dodona
4.  Reading activity isn't marked as read (second item):
![image](https://user-images.githubusercontent.com/56451049/217267796-9dd395b1-c74b-496b-838d-7df4cd37e6e1.png)

"Mark as read" on Dodona itself still works.

# Environment
- OS: Windows 10
- Plugin version: v1.11.0
- ![image](https://user-images.githubusercontent.com/56451049/217267077-19412783-3aaf-4352-b48f-d6f21fc84810.png)

# Screenshots
![image](https://user-images.githubusercontent.com/56451049/217267709-fb6895a0-40cf-4efc-a6b5-118eec862dd2.png)

**Comment by bmesuere** [comment]
We prefer that reading activities can only be marked as completed on Dodona itself.

**Comment by BTWS2** [comment]
@bmesuere what are your concerns about marking reading activities as read via the Dodona extension?

We are looking to solve this issue. Would you be ok with opening the reading activity on Dodona in the default browser instead?

**Comment by bmesuere** [comment]
Since we also use this for exam instructions, we want to be sure that students read the instructions as intended on Dodona where it is impossible to get to the read button without at least opening the page and scrolling to the bottom. 

Opening the reading activity on Dodona, where they can click the button is of course ok.

**Comment by BTWS2** [comment]
Thank you for the feedback. I made a quick mock-up. Is this also ok for you?

I would prefer to keep the option to able to read in VS Code to not disrupt the flow. There can also be a button in the webview to open the page on Dodona in the unlikely event something isn't readable.

![image](https://github.com/user-attachments/assets/a9cab746-55ca-4387-b0f9-6dbd6edd2aab)

**Comment by bmesuere** [comment]
No, we prefer rendering everything on Dodona itself where we test/have control over how everything looks including our javascript that is not part of the description.

**Comment by fejfo** [comment]
If the instructions have to be read on Dodona, you should consider removing the "mark reading activity as complete" button from the extension.
